admiral/pkg/clusters/registry.go

```diff
@@ -42,6 +42,19 @@ func InitAdmiral(ctx context.Context, params common.AdmiralParams) (*RemoteRegis
 		return nil, fmt.Errorf("error with dependency controller init: %v", err)
 	}
 
+	dependencyProxyHandler := DependencyProxyHandler{
+		RemoteRegistry: w,
```

Comment:
Sorry, commenting on an unrelated change: Can we change `w := NewRemoteRegistry(ctx, params)`, and replace `w` with rr`, as we have used `rr` for RemoteRegistry in multiple places.